src/main/java/Duke/Helper/FileCommand.java

```diff
@@ -0,0 +1,62 @@
+package Duke.Helper;
+
+import Duke.Tasks.Deadline;
+import Duke.Tasks.Event;
+import Duke.Tasks.Task;
+import Duke.Tasks.Todo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class FileCommand {
+    public static void loadFile(ArrayList<Task> array) throws IOException {
+        try {
+            File f = new File("data/duke.txt");
+            f.getParentFile().mkdirs();
+            f.createNewFile();
+            Scanner s = new Scanner(f);
+            while (s.hasNext()) {
+                String message = s.nextLine();
+                String[] splitMessage = message.split(" \\| ");
+                String type = splitMessage[0];
+                Integer status = Integer.parseInt(splitMessage[1]);
+                String description = splitMessage[2];
+                switch (type) {
+                case "T":
```

Comment:
Avoid using magic literals such as `"T"`, consider extracting it out into a constant.